src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -825,9 +825,77 @@ public Parent validateParent(Parent parent, Collection<BulkLoadPropertyMappingDT
 							|| foundParent.getCorpName().contains(parent.getLabelPrefix().getLabelPrefix()));
 					if (sameStereoCategory & sameStereoComment
 							& (sameCorpName | (noCorpName & sameCorpPrefixOrNoPrefix))) {
-						// parents match
-						parent = foundParent;
-						break searchResultLoop;
+						// parents match (based on above criteria)
+						boolean equalAliases = true; // assumption is aliases are same unless proven to be false
+
+						// Need to Create String Sets to Do 
```

Comment:
nit: Just because a bunch of the functions in this `BulkLoadServiceImpl` are already quite long, It'd be nice to encapsulate this into a helper function like `private Parent checkAndUpdateParentAliases(Parent parent, Parent foundParent)`. That'll help maintain readability in this already long `validateParent` function.

Reply:
That's fair.

Reply:
@bffrost Comment was marked as resolved but I don't think it actually was, right?

Reply:
Good catch @brianbolt . @dalejerikson  Can you look at pulling this logic out into a helper function just to improve readability?

Reply:
I left an additional separate comment about the break statement that is in the code

Reply:
Implemented